src/test_suite/test_suite.ts

```diff
@@ -493,6 +497,27 @@ export function explainDifferencesCanonical(
     return differences.allok;
 }
 
+export function getYamlInputText(step: TestStep, textType: TextType): string {
```

Comment:
I don't think the behavior of this function is correct. The scenarios are

- TextType.Input: force it to run on the original input, even if there are corrected versions available.
- TextType.STT: if there is correctedSTT available use that otherwise use input, even if there is correctedScope available
- TextType.Scoped: if there is correctedScope available use it, other wise fall back to correctedSTT and then input.

Of these three, we'd like the last one to be the default behavior if the user didn't specify otherwise.

Note that the input we choose to use is a separate feature from filtering and aggregating the results by correction level. You can imagine that we could reserve suite names for these three scenarios and dynamically add these suites as we load the tests.

We could then filter and aggregate by correction level.